app/models/art_item.rb

```diff
@@ -46,23 +47,31 @@ def has_description
       errors.add(:description, "Can't be blank")
     end
   end
+ 
+  private
 
-  def acceptable_documents
-    return unless documents.attached?
-  
-    acceptable_types = [
-      "application/pdf", "text/plain", "image/jpg", 
-      "image/jpeg", "image/png", 
-      "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
-    ]
-
-    documents.each do |doc|
-      unless doc.byte_size <= 20.megabyte
-        errors.add(:documents, "is too big")
+  def validate_documents
+    documents.each do |document|
+      if document.blob.content_type.start_with?('application/') # Check if it's a document
+        if document.blob.byte_size > 10.megabytes # Adjust the size limit as needed
+          errors.add(:documents, "Document size exceeds the 10MB limit.")
+        end
+      else
+        errors.add(:documents, "Invalid file format. Only documents are allowed.")
```

Comment:
What formats are allowed in the validation 'Only documents are allowed'?
I was able to upload .pdf, .docs, .sh, .rtf, but not .txt.
In the form, it says: pdf, txt, doc

Reply:
I updated the instructions for proper doc types

Reply:
I still can upload .sh file

Reply:
set validation only allow pdf, text,doc(x),xls(x), and